src/test/java/racingGameTest/RacingGameTest.java

```diff
@@ -1,19 +1,34 @@
 package racingGameTest;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
-import racingGame.RacingGame;
+import racingGame.domain.RacingGame;
+import racingGame.domain.Winner;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class RacingGameTest {
-    @DisplayName("Test for 'RacingGame', 'moveCar'")
+    private RacingGame racingGame;
+
+    @BeforeEach
+    void setUp() {
+        racingGame = new RacingGame(new String[]{"pobi", "crong", "honux"});
+    }
+
     @Test
-    void racingTest() {
-        int cars = 3;
-        int rounds = 5;
+    void createCarsTest() {
+        String data = "pobi,crong,honux";
+        assertThat(data.split(",").length).isEqualTo(3);    //길이 확인
+        assertThat(data.split(",")).containsExactly("pobi", "crong", "honux");
+    }
 
-        RacingGame racingGame = new RacingGame(cars, rounds);
-        assertThat(racingGame.racing().length).isEqualTo(cars);
+    @DisplayName("Test for 'moveOrNot'")
+    @Test
+    void moveOrNotTest() {
+        assertThat(racingGame.moveOrNot(0)).isEqualTo(false);
```

Comment:
`AbstractBooleanAssert#isTrue()`를 찾아보고 그 기능을 사용해 보면 어떨까요?

```suggestion
        assertThat(racingGame.moveOrNot(0)).isTrue();
```